app/docx/__init__.py

```diff
@@ -0,0 +1,18 @@
+from config import settings
+from constants import DocumentsTypes
+from db.queries import get_decission_data_from_db
+from docx.generate_document import Documents
+from docx.utils import convert_raw_documents_data
+
+
+def generate_documents(record_id: str) -> bool:
```

Comment:
```suggestion
def generate_documents(record_id: str) -> None:
```
I think it's not returning anything.